[MRG] Update: Changed and added Meta Description

TillLord opened the pull request.

betatim commented:
Did you discuss these changes with @kirastina? (Showing Kira how Github works)

Kirastina commented:
He doesn't need to :)

betatim commented on site/content/english/expert-suisse.md:
```diff
@@ -1,6 +1,6 @@
-title: Skribble – Legally binding electronic signatures with Skribble
-description: Easily handle digital signature processes – legally binding in accordance with Swiss & EU law.
+title: Preferential price for Expert-Suisse members
+description: Legally binding electronic signatures with Skribble. Easily handle digital signature processes – legally binding in accordance with Swiss & EU law.
```
```suggestion
description: Legal-ly binding electronic signatures with Skribble. Easily handle digital signature processes – legally binding in accordance with Swiss & EU law.
```

TillLord commented:
Ist es, weil ich ihr Git-Hub zeigen soll, oder weil wir das 4 Augen-Prinzip möchten?

Von: Tim Head <notifications@github.com>
Antworten an: BlockSigner/cove <reply@reply.github.com>
Datum: Montag, 30. September 2019 um 15:06
An: BlockSigner/cove <cove@noreply.github.com>
Cc: Till <till@skribble.com>, Author <author@noreply.github.com>
Betreff: Re: [BlockSigner/cove] [MRG] Update: Changed and added Meta Description (#184)


Did you discuss these changes with @Kirastina<https://github.com/Kirastina>? (Showing Kira how Github works)

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub<https://github.com/BlockSigner/cove/pull/184?email_source=notifications&email_token=AMQPW5Y2ID2PZ56ADCFPYLDQMH2WHA5CNFSM4I3ZVED2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD75SFPI#issuecomment-536552125>, or mute the thread<https://github.com/notifications/unsubscribe-auth/AMQPW5ZZ6EVG75UU6YNEMQDQMH2WHANCNFSM4I3ZVEDQ>.

betatim commented:
We were experimenting with using GitHub.

betatim commented:
I'll merge this now.
